code/game/objects/structures/crates_lockers/closets/phonebox.dm

```diff
@@ -33,18 +34,43 @@
 	anchored = TRUE
 	layer = BETWEEN_OBJECT_ITEM_LAYER
 	exit_stun = 0 //no stun because it's a (glass) 'locker'
```

Comment:
Is it intentional you want no exit stun for this variant? You probably should only specify what values should change for being a `/off` subtype rather than re-applying all values (because then it just undoes changes you do to the base type.

Reply:
yeah I forgot to change this one, should be fixed.